source/infrastructure/disaster-recovery/index.html.md.erb

```diff
@@ -49,13 +49,23 @@ It is possible to lose the database instance and the associated backups. For exa
 |Detection|Endpoint monitoring may point to a healthcheck page checking the connection to the databse. Or smoke tests running in production may detect it.|
 |Remediation|Restore database from external daily or most recent backup|
 
+## Accidental resource deletion
```

Comment:
I wonder whether we could use something like this on DB resources too: https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#prevent_destroy